PetAdoption-iOS/PetAdoption-iOS/Controllers/PetListingDetailVC.swift

```diff
@@ -268,6 +296,31 @@ class PetListingDetailVC: UITableViewController
     }
 
     ////////////////////////////////////////////////////////////
+    
+    
+    ////////////////////////////////////////////////////////////
+    // MARK: - UITableViewDataSource
+    ////////////////////////////////////////////////////////////
+    
+    
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+
+        let sectionNumber = indexPath.section
+        let rowNumber = indexPath.row
+        if sectionNumber == 3 && rowNumber == 1 {
```

Comment:
I get what you're doing here, and this works as the table view is currently set up, but this is probably not the best approach.  You probably want to avoid using "magic numbers" like this.  At the VERY least, I would define constants near the top of the file to make it easier to change these values if the need ever arose, but I actually have a more robust solution for this if you're interested in taking a look.

Reply:
I'd be down to look at the solution you have currently since I'm having issues figuring out how to go about selecting the email row specifically.  I only did it this way for the time being since all the pets have the same number of cells but of course that's subject to change in the future.

Reply:
OK, I'll push up a separate pull request with what I came up with and link it to you on Slack.  I'll be happy to go over with you also if you want.

Reply:
For now, let's just leave this in here like this, only because I don't want you to have to wait on me.  If you could, though, could you add a TODO comment here?  Something like:
```swift
// TODO: Refactor this approach to avoid using magic numbers
```
or something to that effect.